katsdpingest/sigproc.py

```diff
@@ -4,7 +4,8 @@
 
 import pkg_resources
 import numpy as np
-from katsdpsigproc import accel, tune, fill, transpose, percentile, maskedsum, reduce, rfi
+from katsdpsigproc import accel, tune, fill, transpose, percentile, maskedsum, reduce
+import katsdpsigproc.rfi.device
```

Comment:
The changes in this file are not related to the PR - but mypy was giving me grief so I cleaned it up.